packages/react-dom/src/__tests__/ReactTestUtilsAct-test.js

```diff
@@ -731,14 +691,10 @@ function runActTests(label, render, unmount, rerender) {
 
         it('triggers fallbacks if available', async () => {
           if (label !== 'legacy mode') {
-            // FIXME: Support for Blocking* and Concurrent Mode were
-            // intentionally removed from the public version of `act`. It will
-            // be added back in a future major version, before Blocking and and
-            // Concurrent Mode are officially released. Consider disabling all
-            // non-Legacy tests in this suite until then.
-            //
-            // *Blocking Mode actually does happen to work, though
-            // not "officially" since it's an unreleased feature.
+            // FIXME: Support for Concurrent Root intentionally removed
+            // from the public version of `act`. It will be added back in
+            // a future major version, Concurrent Root officially released.
```

Comment:
I think you missed "before" here.

Reply:
Re: "it will be added back" — I did not see it in the pre-RC umbrella. Should it be there?

Reply:
Fixed and done.